src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableEditorEndPoint.class.php

```diff
@@ -1657,13 +1658,13 @@ protected function _WriteDataToDatabase(&$oFields, &$oData, $bDataFromDatabase =
             }
 
             if ($this->bForceHiddenFieldWriteOnSave) {
-                $bAllowChangingField = true;
+                $isFieldChangeAllowed = true;
```

Comment:
For the wording: Is this "any field change allowed" or is this more specific?

Reply:
We only renamed the variable so that it meets the standards and did not change its meaning. But no, it isn't more specific. The variable determines weather or not any field is allowed to be changed (see line 1663)